playbooks/ubuntu.yml

```diff
@@ -1,7 +1,12 @@
 ---
 
 - name: Ubuntu | Install prerequisites
-  raw: sleep 10 && sudo apt-get update -qq && sudo apt-get install -qq -y python2.7
+  raw: "{{ item }}"
```

Comment:
could this be:

```
- name: Ubuntu | Install prerequisites
  apt: name={{ item }} state=latest update_cache=yes
  with_items:
    - python2.7
    - sudo
```

Reply:
Not really. Ubuntu supplies with python3 by default and we don't want to take such risk. Let me try to change `ansible_python_interpreter` to python3 for these particular tasks.